src/types/runtime/index.ts

```diff
@@ -236,6 +237,11 @@ export interface Runtime {
 
   exports: RuntimeExports;
 
+  /**
+   * An optional filesystem that can be added by a plugin
+   */
+  fs?: NotebookFilesystem;
```

Comment:
I'm thinking that perhaps this should be an object that is always present, that allows one to mount under a prefix. 

I'm a bit anxious about this change: this is something we can't change in the future without breaking existing notebooks. You could put this under `internal` to indicate people shouldn't mess with this themselves? Or we don't put it on the runtime object at all yet, and instead put it in a plugin's exports.

Reply:
I'll put it in `internal` for now